docs/development/hatch.rst

```diff
@@ -0,0 +1,257 @@
+commands that are not present in any other environment declared above. For
+executing the command you have to follow this simple formula::
+
+  $ hatch run <environment name>:<command name>
+
+Then applied to the ``push`` command on the ``transifex`` environment will be::
+
+  $ hatch run transifex:pull
+
+You can use the same logic for execute the available commands in the app, but
+heres a detailed list of the commands ordered by environments, so you can
+understand the purpose of each one::
+
+* ``cov``: Check coverage status.
+
+* ``no-cov``: Check places pending to add coverage.
+
+* ``lint``: Make sure the code changes follow our guidelines and conventions.
+
+* ``clean-build``: Remove the files generated after the project is built.
+
+* ``clean-pyc``: Remove pyc generated files.
+
+* ``assets``: Generate the static files used by django-wiki frontend.
+
+* ``test``: Test the changes in the current environment.
+
+* ``test:all``: Test the changes across our supported Python and Django versions.
+
+* ``test:lint``: Make sure the code changes follows our guidelines and conventions.
+
+* ``test:clean``: Remove the files generated via the testing process.
+
+* ``transifex:push``: Push the translation files to Transifex.
+
+* ``transifex:pull``: Pull the translation files from Transifex.
+
+* ``docs:clean``: Remove the generated documentation files.
+
+* List of docs commands used to generate the documentation in different formats:
+
+  * Please refer to the `Builder documentation of SPHINX <https://www.sphinx-doc.org/en/master/usage/builders/index.html>`_
+    to understand more about the purpose of each builder and the expected output.
+
+  * ``docs:html``
+
+  * ``docs:dirhtml``
+
+  * ``docs:singlehtml``
+
+  * ``docs:pickle``
+
+  * ``docs:json``
+
+  * ``docs:htmlhelp``
+
+  * ``docs:qthelp``
+
+  * ``docs:devhelp``
+
+  * ``docs:epub``
+
+  * ``docs:latex``
+
+  * ``docs:latexpdf``
+
+  * ``docs:text``
+
+  * ``docs:man``
+
+  * ``docs:texinfo``
+
+  * ``docs:info``
+
+  * ``docs:gettext``
+
+  * ``docs:changes``
+
+  * ``docs:link-check2``
+
+  * ``docs:doctest``
+
+
+* ``docs:build``: Generate the documentation in HTML format.
+
+* ``docs:link-check``: Checks for external links across the documentation.
+
+We hope that this document helps you to understand more about the development
+process, if something is not clear please open an issue.
+
+FAQ
+---
+
+1. **Whats the difference between test and test:all?**
+
+   When you execute ``hatch run test`` this will check your changes in the
+   active environment, this means it will run over an specific Python version
+   and an specific Django Version; in the other hand ``test:all`` will run the
+   test suite in the whole matrix of the supported versions of Python and Django.
+
+2. **What Python Versions are currently used?**
+
+   Right now, ``django-wiki`` runs on Python 3.7.X until 3.10.X and for Django
+   it runs in version 2.2 until version 4.0.
```

Comment:
I think this section should be removed since it's mentioned in the main README.. so the above information will be hard to maintain. Likely, it will be inaccurate 1 year from now :)